staking/types/validator.go

```diff
@@ -95,7 +95,7 @@ type ValidatorWrapper struct {
 type Computed struct {
 	Signed            *big.Int    `json:"current-epoch-signed"`
 	ToSign            *big.Int    `json:"current-epoch-to-sign"`
-	BlocksLeftInEpoch uint64      `json:"current-epoch-blocks-left"`
+	BlocksLeftInEpoch uint64      `json:"num-beacon-blocks-until-next-epoch"`
```

Comment:
why is this name change? from `current-epoch-blocks-left` to `num-beacon-blocks-until-next-epoch`? I don't like the new name.

Reply:
Changing the name to reflect the new meaning of the field.
Blocks left per epoch doesn't mean anything for non-beacon chain, since the number of blocks on auxiliary shards are dependent on the speed of beacon chain relative to the other shards.
Calculating number of blocks left on Beaconchain until next epoch instead of how many left depending on blocks signed by the validator.

Reply:
would this impact staking explorer? @mikedoan do you agree on this change?

Reply:
generally, we shouldn't change the API like naming once we have downstream users like dashboard/explorer unless consumers agreed and made corresponding changes.

Reply:
this field is just an informational field for validators. It's not used for any downstream programs like UI yet.